### ISSUE TITLE
FATA[0000] unknown logging driver "syslog": not found  

### ISSUE THREAD

**Issue by Sam-Sundar-Apty** [containerd/nerdctl#1337]
### Description

Hey Team,

We are trying to setup harbor registry with nerdctl compose and while spinning up the same, we got CNI errors which were fixed by solutions from [this](https://github.com/containerd/nerdctl/discussions/1129). 
Now we are getting below error,
`FATA[0000] unknown logging driver "syslog": not found  ` 

### Steps to reproduce the issue

1.
2.
3.

### Describe the results you received and expected

With docker and docker-compose these errors never occured.

### What version of nerdctl are you using?

ubuntu@ip-172-16-4-239:~/harbor/harbor$ nerdctl --version
`nerdctl version 0.22.2`

### Host information

sudo nerdctl info
```
Client:
 Namespace:	default
 Debug Mode:	false

Server:
 Server Version: 1.4.13
 Storage Driver: overlayfs
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Log: fluentd journald json-file
  Storage: native overlayfs
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 5.11.0-1022-aws
 Operating System: Ubuntu 20.04.4 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 16
 Total Memory: 29.36GiB
 Name: ip-172-16-4-239
 ID: eb0871bf-d4a3-4762-ba89-43f1d7599dec
```

**Comment by AkihiroSuda** [comment]
The `syslog` driver is unimplemented. Please use `json-file`, `fluentd`, or `journald`

**Comment by fahedouch** [comment]
Added to the list of drivers https://github.com/containerd/nerdctl/issues/1039

**Comment by yardenshoham** [comment]
Is this closed since #1377 was merged?
